### DISCUSSION TITLE
RTL mode isn't compatible with Laravel Octane

### DISCUSSION THREAD

**Discussion by NargesHajiani** [laravel/nova-issues#4485] [Q&A] [answered]
- Laravel Version: 9.18.0
- Nova Version: 4.10.0
- PHP Version: 8.1.0
- Octane Version: 1.2.12
- Database Driver & Version: MariaDB-1:10.6.5
- Operating System and Version: Ubuntu 20.0.4
- Browser type and version: Chome 102.0.5005.115

### Description:
RTL mode isn't compatible with Laravel Octane and `Nova::enableRTL()` has no effect on panel layout.

### Detailed steps to reproduce the issue on a fresh Nova installation:
- Just put `Nova::enableRTL()` in `NovaServiceProvider` boot method.

**Comment by crynobone** [comment, marked as answer]
In Octane, all service providers are only loaded on the initial request. To handle it for Octane wrapped it under `Nova::serving()` to execute it for each request
